get-started/part4.md

```diff
@@ -108,6 +109,17 @@ to open a terminal session on that VM. Type `exit` when you're ready to return
 to the host shell prompt. It may be easier to paste the join command in that
 way.
 
+You can `ssh` into your manager (`myvm1`), and type `docker node ls`:
```

Comment:
I know, it's commonly used. Just like "to grep" as a verb. But it's a nightmare for people who have a different native language than English, and it causes a lot of problems in translation.